services/leecher/leecher/listener.py

```diff
@@ -38,8 +38,6 @@ def __init__(self, func: Union[Callable, None] = None):
         else:
             self.func = func
 
-        logging.debug(f"Callback method is {self.func}.")
```

Comment:
why are you removing all the debug logs? that's the point of them that they should just be for debugging? can you not change the logging config of the server to choose if you don't want to see them?

Reply:
shouldn't we then just finish https://github.com/ynput/ayon-shotgrid/pull/95 and revert the removal of the debug calls here?